target/linux/ramips/image/mt76x8.mk

```diff
@@ -384,6 +384,18 @@ define Device/jotale_js76x8-32m
 endef
 TARGET_DEVICES += jotale_js76x8-32m
 
+define Device/keenetic_kn-1212
+  BLOCKSIZE := 64k
+  IMAGE_SIZE := 15073280
+  DEVICE_VENDOR := Keenetic
+  DEVICE_MODEL := KN-1212
+  DEVICE_PACKAGES := kmod-mt7603 kmod-usb2
```

Comment:
kmod-mt7603 is [default package for mt76x8](https://github.com/openwrt/openwrt/blob/main/target/linux/ramips/mt76x8/target.mk#L10)